autoprimenet.py

```diff
@@ -4814,9 +4814,19 @@ def parse_prpll_log_file(adapter, adir, p):
 	return iteration, avg_msec_per_iter, stage, pct_complete, fftlen, bits, buffs
 
 
-def parse_mfaktc_output_file(adapter, adir, p):
+def get_mfaktc_output_filename(adir, p, sieve_depth, factor_to):
+	"""Get the mfaktc output filename based on the assignment and mfaktc version."""
+	filenames = glob.glob("M{0}_{1}-{2}_*.ckp".format(p, int(sieve_depth), int(factor_to)))
+	for filename in filenames:
+		if (os.path.isfile(os.path.join(adir, filename))):
+			return filename
```

Comment:
The `os.path.join()` would need to be done as part of the `glob.glob()` call:
```suggestion
	filenames = glob.glob(os.path.join(adir, "M{0}_{1}-{2}_[0-9]*.ckp".format(p, int(sieve_depth), int(factor_to))))
	if filenames:
		return os.path.basename(filenames[0])
```